Version.targets

```diff
@@ -0,0 +1,62 @@
+<Project>
```

Comment:
this is probably not required once we drop GenerateAssemblyInfo and the special-cases for CI